micromasters/utils.py

```diff
@@ -19,6 +19,14 @@ def webpack_dev_server_url(request):
     return 'http://{}:{}'.format(webpack_dev_server_host(request), settings.WEBPACK_DEV_SERVER_PORT)
 
 
+def remove_key(dictionary, key):
+    """
+    Helper method to remove a key from a dict and return the dict
+    """
+    del dictionary[key]
+    return dictionary
```

Comment:
docstring looks good to me ("Helper method to remove a key from a dict and return the dict"). describes exactly what it's doing. function name is now `dict_without_key` - does that help?